packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -208,7 +208,7 @@ const VisitNotesForm: React.FC<VisitNotesFormProps> = ({ patientUuid, isTablet }
   );
 
   const handleSubmit = React.useCallback(
-    (event: SyntheticEvent<HTMLFormElement>) => {
```

Comment:
Is there a reason for removing the type? Having a type here is better then an implicit `any` imo.

Reply:
It was giving an error in this file, actually I think there's an update in carbon's Button, because I'm encountering many errors in management repo too.

Reply:
Probably not in Carbon's Button directly, but we no longer import the components from the `.../es/...` submodules which results in us suddenly having TypeScript typings for the components. -> TS can now complain in locations where we had no typings before.
I myself am not getting an error in this location. Maybe you could try pulling from master/updating packages if the error persists on your side. The issue may also just be that the `HTMLFormElement` type conflicts with the button's `onClick` handler type. As a quick fix, you could replace `SyntheticEvent<HTMLFormElement>` with `SyntheticEvent` since the `event.preventDefault` call is the only thing required by the handler.
In any case, fixing the typing instead of removing them is, whenever possible, better.